components/services/brave_wallet/public/proto/zcash_grpc_data.proto

```diff
@@ -4,6 +4,15 @@ option optimize_for = LITE_RUNTIME;
 
 package zcash;
 
+message TreeState {
```

Comment:
this seems to be copied from https://github.com/zcash/lightwalletd/blob/c6a3fe1574136eb782aebd6ee652b4d825925e60/walletrpc/service.proto#L121 ? Why aren't we just including the original?

Reply:
Do you suggest just to copy or to include another third-party dependency?